src/NServiceBus.Core/Outbox/OutboxSendBehavior.cs

```diff
@@ -15,7 +15,15 @@ public void Invoke(OutgoingContext context, Action next)
 
             if (context.TryGet(out currentOutboxMessage))
             {
-                currentOutboxMessage.TransportOperations.Add( new TransportOperation(context.OutgoingMessage.Id, context.DeliveryOptions.ToTransportOperationOptions(), context.OutgoingMessage.Body, context.OutgoingMessage.Headers)); 
+                var options = context.DeliveryOptions.ToTransportOperationOptions();
+                var transportOperation = new TransportOperation(context.OutgoingMessage.Id, options, context.OutgoingMessage.Body, context.OutgoingMessage.Headers);
+
+                if (context.OutgoingMessage.TimeToBeReceived != TimeSpan.MaxValue)
+                {
+                    options["TimeToBeReceived"] = context.OutgoingMessage.TimeToBeReceived.ToString();
```

Comment:
OK, MSDN says `Timespan.ToString` remains culture-insensitive in .NET 4 (https://msdn.microsoft.com/en-us/library/1ecy8h51(v=vs.110).aspx). So I am good with merge.